docs/docs/recce-cloud/setup-gh-actions.md

```diff
@@ -119,6 +119,16 @@ jobs:
 If executed successfully, the dbt target will be placed in the run artifacts and will be named **target**.
 ![alt text](../../assets/images/recce-cloud/dbt-artifacts.png){: .shadow}
 
+### Upload base artifacts to Recce Cloud
```

Comment:
I think but not sure if this is the best place to put this part of doc.

Reply:
Yes, I think we can talk about uploading base artifacts to Recce Cloud here since it's right after the base workflow section.

Reply:
So depending on the above first comment, here should be the #3 

1. the whole workflow? like below: 
   Fore dbt Core users, this workflow will perform the following actions:
   1. Run dbt on the base environment.
   2. Upload the generated DBT artifacts to [github workflow artifacts](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts) for later use.
   3. If executed successfully, the dbt target will be placed in the run artifacts and will be named target.
 
   For dbt Cloud users, this workflow will perform the following actions:
   1. ?? 
   2. ??
   3. ??
4. only the dbt artifacts part? 
   This workflow will perform the following actions:
   1. Run dbt on the base environment.
   2. For dbt Core users, 
      - upload the generated DBT artifacts to [github workflow artifacts](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts) for later use.
      - If executed successfully, the dbt target will be placed in the run artifacts and will be named target.
   5. For dbt Cloud users, we will provide another github workflow artifacts soon.
      - ??
      - ??
  
https://datarecce.io/assets/images/recce-cloud/dbt-artifacts.png is for dbt Cloud?